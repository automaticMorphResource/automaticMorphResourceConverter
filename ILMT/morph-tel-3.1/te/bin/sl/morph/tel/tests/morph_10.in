1	A	unk	
1	cepala	unk	
1	raMgulu	unk	
1	,	unk	
1	vAti	unk	
1	SarIraMpE	unk	
1	uMde	unk	
1	milamilalu	unk	
1	,	unk	
1	dijEnlanu	unk	
1	kalYlArA	unk	
1	cUdavalasiMxe	unk	
1	wappa	unk	
1	mAtallo	unk	
1	mAwraM	unk	
1	varNiMceMxuku	unk	
1	vIlukAxu	unk	
1	.	unk	
