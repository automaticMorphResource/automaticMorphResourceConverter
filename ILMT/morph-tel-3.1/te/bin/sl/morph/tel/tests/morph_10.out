1	A	unk	<fs af='A,avy,,,,,,'>|<fs af='A,avy,,,,,,'>
2	cepala	unk	<fs af='cepa,n,,pl,,o,ti,ti'>
3	raMgulu	unk	<fs af='raMgu,n,,pl,,d,0,0'>|<fs af='raMgu,n,,pl,,d,0,0'>|<fs af='raMgu,n,,pl,,,vu,vu'>|<fs af='raMgu,n,,pl,,,vu,vu'>|<fs af='raMg,n,,pl,,d,0,0'>|<fs af='raMg,n,,pl,,,vu,vu'>|<fs af='raMgi,n,,pl,,d,0,0'>
4	,	unk	<fs af='&comma;,punc,,,,,,'>
5	vAti	unk	<fs af='axi,pn,fn,pl,3,o,ti,ti'>
6	SarIraMpE	unk	<fs af='SarIraM,n,,sg,,,pE,pE'>
7	uMde	unk	<fs af='uMdu,v,any,any,any,,e,e'>|<fs af='uMdu,v,any,any,any,,i,i_e'>
8	milamilalu	unk	<fs af='milamila,n,,pl,,d,0,0'>|<fs af='milamila,n,,pl,,,vu,vu'>
9	,	unk	<fs af='&comma;,punc,,,,,,'>
10	dijEnlanu	unk	<fs af='dijEn,n,,pl,,,ni,ni'>
11	kalYlArA	unk	<fs af='kalYlArA,n,,sg,,d,0,0'>|<fs af='kalYlArA,n,,sg,,o,ti,ti'>
12	cUdavalasiMxe	unk	<fs af='cUdu,v,fn,sg,3,,a_valayu_A,a_valayu_A_e'>
13	wappa	unk	<fs af='wappa,avy,,,,,0,0_avy'>|<fs af='wappa,n,,sg,,d,0,0'>|<fs af='wappa,n,,sg,,o,ti,ti'>|<fs af='wappu,v,any,any,any,,an,an'>
14	mAtallo	unk	<fs af='mAta,n,,pl,,,lo,lo'>
15	mAwraM	unk	<fs af='mAwraM,avy,,,,,,'>|<fs af='mAwraM,avy,,,,,0,0_avy'>
16	varNiMceMxuku	unk	<fs af='varNiMcu,v,any,any,any,,e_eMxuku,e_eMxuku'>
17	vIlu	unk	<fs af='vIlu,n,,sg,,d,0,0'>|<fs af='vIlu,n,,sg,,o,ti,ti'>|<fs af='vIl,n,,sg,,d,0,0'>|<fs af='vIl,n,,sg,,o,ti,ti'>|<fs af='vI,n,,pl,,d,0,0'>|<fs af='vI,n,,pl,,,vu,vu'>
17	kAxu	unk	<fs af='kAxu,avy,,,,,0,0_avy'>|<fs af='avvu,v,fn,sg,3,,a,a'>
18	.	unk	<fs af='&dot;,punc,,,,,,'>
